create_llama_swap_list.py

```
@@ -0,0 +1,92 @@
+				elif 'Q' in file.upper():
+					# Fallback logic to grab string between dots/dashes
+					parts = re.split(r'[\.\-_]', file)
+					for p in parts:
+						if p.upper().startswith('Q') and any(
+							char.isdigit() for char in p
+						):
+							found_quants.add(p.upper())
+
+		return sorted(list(found_quants))
+
+	except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)